tests/unittest_builder.py

```diff
@@ -614,7 +615,9 @@ def test_module_base_props(self) -> None:
         self.assertEqual(module.package, 0)
         self.assertFalse(module.is_statement)
         self.assertEqual(module.statement(), module)
-        self.assertEqual(module.statement(), module)
+        self.assertEqual(module.statement(False), module)
```

Comment:
After the recent change `statement(False)` should never be called. Either `None` or `True`.